demo/src/screens/componentScreens/GridListScreen.tsx

```diff
@@ -0,0 +1,60 @@
+import React, {Component} from 'react';
+import {StyleSheet} from 'react-native';
+import {View, Text, Constants, GridList, Card, Spacings, BorderRadiuses, GridListProps} from 'react-native-ui-lib';
+import products from '../../data/products';
+
+class GridListScreen extends Component {
+  state = {
+    orientation: Constants.orientation
+  };
+
+  renderItem: GridListProps<typeof products[0]>['renderItem'] = ({item}) => {
+    return (
+      <Card flex>
```

Comment:
Consider adding `$backgroundElevated` for the card background and `$textDefault` for the texts